test/common.js

```diff
@@ -20,6 +20,13 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+// Create tmpDir if it does not exist
+try {
+  fs.accessSync(exports.tmpDir);
```

Comment:
My thinking was that if there is no temp directory and creating the temp directory failed, I wanted the CLI to blow up.